### ISSUE TITLE
Voice over

### ISSUE THREAD

**Issue by MichaelaRob** [nilababaei/ecommerce-pattern-library#4]
Fix voiceover controls (aria details etc) description on items missing with voiceover

**Comment by nilababaei** [comment]
Thank you, got it fixed up!
